homeassistant/components/system_bridge/__init__.py

```diff
@@ -162,6 +173,16 @@ async def handle_open_path(call: ServiceCall) -> None:
             OpenPath(path=call.data[CONF_PATH])
         )
 
+    async def handle_power_command(call: ServiceCall) -> None:
+        """Handle the power command service call."""
+        _LOGGER.info("Power command: %s", call.data)
+        coordinator: SystemBridgeDataUpdateCoordinator = hass.data[DOMAIN][
+            call.data[CONF_BRIDGE]
+        ]
+        await getattr(
+            coordinator.websocket_client, f"power_{call.data[CONF_COMMAND]}"
```

Comment:
Even though there is a validation, is it a good practice to concatenate like this, any risk of arbitrary code execution ?